stix_shifter_modules/aws_guardduty/stix_translation/results_translator.py

```diff
@@ -0,0 +1,5 @@
+from stix_shifter_utils.stix_translation.src.json_to_stix.json_to_stix import JSONToStix
+
+
+class ResultsTranslator(JSONToStix):
+    pass
```

Comment:
this file doesn't do anything. you can just remove `results_translator.py`

Reply:
Removed this file here and the changes are also present in AWS GuardDuty PR #1525